.circleci/config.yml

```diff
@@ -1,10 +1,18 @@
-version: 2
-
-references:
-  container_config: &container_config
+version: 2.1
+executors:
+  infrastructure_container_unstable:
+    docker:
+      - image: aeternity/infrastructure:master
+  infrastructure_container_stable:
+    docker:
+      - image: aeternity/infrastructure:stable
```

Comment:
Is a note anywhere re semantics of stable vs. master branches of infrastructure repo?

Reply:
Nope, but stable is meant to replace the tagging